src/connector/src/sink/iceberg/mod.rs

```diff
@@ -2031,7 +2049,7 @@ impl IcebergSinkCommitter {
         Ok(Some((write_results, snapshot_id)))
     }
 
-    async fn commit_iceberg_inner(
+    async fn commit_datafile(
```

Comment:
These renamings will make the implementation of 2PC clearer later, so I included them in this pull request.